src/buildCommon.js

```diff
@@ -222,10 +222,10 @@ const boldsymbol = function(
 /**
  * Makes either a mathord or textord in the correct font and color.
  */
-const makeOrd = function<NODETYPE: "textord" | "mathord">(
+const makeOrd = function<NODETYPE: "spacing" | "textord" | "mathord">(
     group: ParseNode<NODETYPE>,
     options: Options,
-    type: NODETYPE,
+    type: NODETYPE | "textord",
```

Comment:
Isn't `"textord"` already included in `NODETYPE`?

Reply:
You're right, that's confusing. I changed `type` to `"mathord" | "textord"` instead. Previous PR dealt with the case where `ParseNode<"mathord">` was called type `"mathord"` and similarly for `"textord"` so the original type made sense. Now, I've got `ParseNode<"spacing">` is called with type `"textord"` which means that I have to break the tight-knit relationship between the `ParseNode` type and `type`.